src/main/java/hw5/WebDriverSingletone.java

```diff
@@ -0,0 +1,27 @@
+package hw5;
+
+import org.openqa.selenium.WebDriver;
+
+import java.util.Objects;
+
+public class WebDriverSingletone {
```

Comment:
Just like the whiskey
```suggestion
public class WebDriverSingleton {
```